### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by scottinet**
# Description

Update/modernize the CONTRIBUTING.md file

# Other changes

Expose Kuzzle's MQTT port to allow functional tests to be run from outside docker images.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,17 +35,22 @@ How to run the development stack (needs Docker 1.10+ and Docker Compose 1.8+):
...
-git submodule init
-git submodule update
-
-# start kuzzle with development tools enabled
+# start a kuzzle cluster with development tools enabled
 docker-compose up
 ```
 
-You can now access to `http://localhost:7512` for the standard Kuzzle HTTP, WebSocket and MQTT APIs
+You can now access to the Kuzzle HTTP/WebSocket API through the following URL: `http://localhost:7512`.
```
Comment (Leodau) [inline CONTRIBUTING.md:42]:
```suggestion
You can now access the Kuzzle HTTP/WebSocket API through the following URL: `http://localhost:7512`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -78,21 +84,18 @@ See our [plugins documentation](https://docs.kuzzle.io/core/2/plugins/)
 ```bash
 $ docker-compose up -d
 
-# wait for Kuzzle stack to be up
+# wait for Kuzzle stack to be up, and start the entire test suite (long)
```
Comment (Leodau) [inline CONTRIBUTING.md:87]:
```suggestion
# Wait for Kuzzle stack to be up, and start the entire test suite (long)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -34,17 +35,22 @@ How to run the development stack (needs Docker 1.10+ and Docker Compose 1.8+):
 git clone git@github.com:kuzzleio/kuzzle.git
 cd kuzzle
 
-# don't forget to retreive default plugins embeded in submodules
-git submodule init
-git submodule update
-
-# start kuzzle with development tools enabled
+# start a kuzzle cluster with development tools enabled
```
Comment (Leodau) [inline CONTRIBUTING.md:38]:
```suggestion
# Start a kuzzle cluster with development tools enabled
```

**Review by Leodau — CHANGES_REQUESTED** [review]
Just some anti OCD for me.

**Review by Leodau — APPROVED** [review]

**Review by Aschen — APPROVED** [review]

**Pull request merged**